normandy/control/views.py

```diff
@@ -0,0 +1,18 @@
+from django.views import generic
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from normandy.recipes.models import Recipe
+
+
+class IndexView(LoginRequiredMixin, generic.ListView):
```

Comment:
How do we feel about using these built-in generic views (like generic.ListView)? Are they going to be too limiting for us? Tough to tell how restrictive they are at first glance.